commands/miscellaneous/getUsersInServer.js

```diff
@@ -1,73 +1,73 @@
 const Discord = require('discord.js');
 const { SlashCommandBuilder } = require('@discordjs/builders');
-const { BATId, BALId, BAGId } = require('./config.json');
+const { BATId, BALId, BAGId } = require('../../config.json');
```

Comment:
This was the main change, the file path was incorrect. The rest of the changes was changing the spacing from 4 spaces to 2 spaces.